app/config/config.yml

```diff
@@ -43,6 +43,7 @@ framework:
 
 # Twig Configuration
 twig:
+    default_path: '%kernel.project_dir%/templates'
```

Comment:
here where the magic happen, from https://github.com/symfony/recipes/blob/main/symfony/twig-bundle/5.4/config/packages/twig.yaml